test/acceptance/features/immutabeSuccessfulBinding.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Successful Service Binding Resource should be Immutable 
+
+    As a user of Service Binding operator
+    I should not be able to apply changes of an already appled and successful SBR
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+        * CustomResourceDefinition backends.stable.example.com is available
+
+    Scenario: Forbid user from reapplying the SBR for an already succesful SBR
+        Given The Custom Resource is present
+        """
+        apiVersion: stable.example.com/v1
+        kind: Backend
+        metadata:
+            name: service-immutable
+            annotations:
+                service.binding/host: path={.spec.host}
+        spec:
+            host: foo
+        """
+        And Generic test application "app-immutable" is running
+        When Service Binding is applied
```

Comment:
Use `And` here, because this is still test setup. We should have only one `When` and `Then` block

Reply:
Done